src/pudl/constants.py

```diff
@@ -245,7 +245,7 @@
     'no. 2  oil', 'blended oil', 'no 2. oil', '# 6 oil', 'nno. 2 oil',
     '#2 fuel', 'oill', 'oils', 'gas/oil', 'no.2 oil gas', '#2 fuel oil',
     'oli', 'oil (#6)', 'oil/diesel', '2 oil', '#6 hvy oil', 'jet fuel',
-    'diesel/compos', 'oil-8', 'oil {6}', 'oil-unit #1', 'bbl.', 'oil.',
+    'diesel/compos', 'oil-8', 'oil {6}', 'oil-unit #1', 'bbl.', 'oil.',  # noqa: FS003
```

Comment:
why? that is the issue here? is it the {'s?

Reply:
Yes, the `flake8-use-fstring` linter misinterprets the `"oil {6}"` string literal as an f-string without an f-prefix.

Reply:
got it! thanks